src/CommandLineParser.h

```diff
@@ -72,6 +72,14 @@ class CommandLineParser
      */
     bool contains(const QString & key) const;
 
+    /**
+     * returns true if the option was found.
+     * Consider the following definition "--expert level"  The user can type as an argument
+     * "--expert 10".  Calling contains("expert") will return true.
+     * @see optionArgument()
+     */
+    bool contains(const QLatin1String & key) const;
```

Comment:
Here, either we add a contains with a QLatin1String or we change everywhere to use a QStringLiteral instead of a QLatin1String. From what I understand, as there are overrides for QLatin1String  in QListString.contains, it should be faster to have an override instead of using QStringLiteral